charts/supabase/templates/db/statefulset.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.db.enabled -}}
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
```

Comment:
This looks like a polluted PR

Reply:
I've created a clean PR that replaces this.